packages/core/src/browser/view-container.ts

```diff
@@ -46,6 +47,9 @@ export interface ViewContainerTitleOptions {
     closeable?: boolean;
 }
 
+const SASH_DELAY_PREFERENCE = 'workbench.sash.hoverDelay';
+const SASH_SIZE_PREFERENCE = 'workbench.sash.size';
```

Comment:
I don't believe the variables are necessary, the `CorePreferences` should already know about them.